content/200-concepts/100-components/02-prisma-client/059-embedded-types.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: 'Embedded documents'
+metaTitle: 'Embedded documents (Preview)'
+metaDescription: 'This page explains how to use embedded documents.'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+<Admonition type="info">
+
+Embedded documents are available in Preview in versions 3.10.0 and later.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Embedded documents are currently only available on MongoDB.
+
+</Admonition>
+
+Embedded documents introduce a new way of linking records in Prisma. There are two ways to establish a link between two records:
+
+- **Creating a relation**, by using a foreign key to link from one record to another record.
+- **Embedding a record**, by storing a record within another record.
+
+Prisma has great [support for relations](/concepts/components/prisma-schema/relations), but support for embedded data has previously been limited to the following options:
+
+- Using [scalar lists](/concepts/components/prisma-client/working-with-fields/working-with-scalar-lists-arrays) to store multiple values in a single field. Only lists of scalar fields are supported.
+- Using [`Json` fields](/concepts/components/prisma-client/working-with-fields/working-with-json-fields) to store JSON data within a single field. `Json` fields are untyped.
+
+Embedded documents introduce a new way to embed data:
+
+- **Embedded documents** allow you to define new object types. Embedded documents are structured and typed, similar to how models are structured and typed today. By understanding the contents of the embedded data, we can offer a richer set of operations you can perform on that data.
```

Comment:
nit: Feels a bit weird having a list with one item.

Reply:
This got removed when I rearranged the information